src/pages/MyProfile.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import ProfileMissions from '../components/ProfileMissions';
+import RocketProfile from '../components/RocketProfile';
+import styles from './MyProfile.module.css';
+
+const MyProfile = () => {
+  const missions = useSelector((state) => state.missions);
+  const profileMission = missions.filter((mission) => mission.reserved === true);
+  const rockets = useSelector((state) => state.rockets);
+  const reservedRockets = rockets.filter((rocket) => rocket.reserved === true);
+  return (
```

Comment:
Awesome 🔥 You are rendering a list of reserved rockets and missions without any issue!